fxdc/parsedata/lexer.py

```
@@ -68,6 +70,9 @@ def __repr__(self) -> str:
 class Lexer:
     def __init__(self, text: str, classes: list[str] = []) -> None:
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)